README.md

```diff
@@ -74,8 +98,23 @@ yarn start
 
 <!-- TODO: document this -->
 
+
+### Scope of the project
+
+For now scope of the project includes:
+
+- map with markers of places
+- list of places of interest by type and category
+- views and UIs that comply with Helsinki City brand guidelines
+- localization
+- Test experience views
+- accesibility
+- server side rendering
+- visual and programmatic testing
```

Comment:
> visual

e2e testing is something that should also be added to the new repo.

Reply:
Thanks!